src/fides/api/models/privacy_request.py

```diff
@@ -957,7 +964,7 @@ def get_cached_identity_data(self) -> Dict[str, Any]:
     def verify_identity(
         self,
         db: Session,
-        provided_code: str,
+        provided_code: str = None,
```

Comment:
```suggestion
        provided_code: Optional[str] = None,
```

I don't think the current type makes sense technically? Not sure why `mypy` isn't catching it. An alternative is also

`provided_code: str = "",`

Reply:
I left it as `None` explicitly to handle the case where the method is called like this:

```
verify_identity(db, provided_code=None)

# or
code = None
verify_identity(db, provided_code=code)
```

Reply:
are there cases where the method is called like that? That still doesn't seem right...

If it is invoked as `provided_code=None` that's not correct from a type standpoint. An empty string also evaluates to `False` in a truthy check so `provided_code: str = ""` makes sense if we want to let it default to a falsey value?

Reply:
This is the consent request verification method — there are several existing places in the codebase this is called with a `None` value:
- https://github.com/ethyca/fides/blob/main/src/fides/api/api/v1/endpoints/consent_request_endpoints.py#L283
- https://github.com/ethyca/fides/blob/main/src/fides/api/api/v1/endpoints/consent_request_endpoints.py#L414